apps/sms/views/inbox/js/inbox.js

```diff
@@ -765,8 +765,8 @@ var InboxView = {
     li.innerHTML = this.tmpl.thread.interpolate({
       hash: isDraft ? '#/composer' : '#/thread?id=' + id,
       mode: isDraft ? 'drafts' : 'threads',
-      id: isDraft ? draft.id : id,
-      number: number,
+      id: String(isDraft ? draft.id : id),
```

Comment:
sorry, this has a conflict with current master; you just need to rebase abd add this change back to the new code (`number` is gone :) )